src/sentry/static/sentry/app/views/organizationIntegrations/integrationRow.tsx

```diff
@@ -120,4 +132,17 @@ const PublishStatus = styled(({status, ...props}: PublishStatusProps) => (
   }
 `;
 
+const CategoryTag = styled(({category, ...p}) => <div {...p}>{category}</div>)`
+  display: flex;
+  flex-direction: row;
+  padding: 1px 10px;
+  background: ${p => (p.category === p.publishStatus ? '#E1567C' : p.theme.offWhite2)};
```

Comment:
@NisanthanNanthakumar Is there no theme color for `#E1567C`? Can you check with someone from design?

Reply:
@scefali Checked with @doralchan. Using purpleLightest from the theme instead.